### PR TITLE
docs: readme pass

### PR DESCRIPTION
**Pull request by onbjerg**
Reformats the README slightly and adds missing information (e.g. links to crate docs)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,15 +35,40 @@ More concretely, our goals are:
 
 ## Status
 
-The project is not ready for use. We hope to have full sync implemented sometime in Q1 2023, followed by optimizations. In the meantime, we're working on making sure every crate of the repository is well documented, abstracted and tested.
+The project is **not ready for production use**.
+
+We hope to have full sync implemented sometime in Q1 2023, followed by optimizations. In the meantime, we're working on making sure every crate of the repository is well documented, abstracted and tested.
```
Comment (mattsse) [inline README.md:40]: I think this is no longer up to date, since Q1 has passed and we also are able to fully sync

**Review by mattsse — APPROVED** [review]
lgtm overall,

left one q re full sync timeline

File: README.md [hunk 2]
```diff
@@ -33,15 +35,38 @@ More concretely, our goals are:
...
+
+It has **not been audited for security purposes** and should not be used in production yet.
+
+We will be updating the documentation with the completion status of each component, as well as include more contributing guidelines (design docs, architecture diagrams, repository layouts) and "good first issues".
+
+We appreciate your patience until we get there. Until then, we are happy to answer all questions in the Telegram link above.
+
+## For Users
+
+See the [Reth Book](https://paradigmxyz.github.io/reth) for instructions on how to install and run Reth.
```
Comment (rkrasiuk) [inline README.md:50]: it'd be really nice to have book and docs links somewhere at the beginning of the readme. usually, the first thing i look for

**Review by rkrasiuk — APPROVED** [review]
lgtm

**Review by mattsse — APPROVED** [review]

**Pull request merged**